rebar.config

```diff
@@ -1,20 +1,130 @@
+%% -*- mode: erlang; erlang-indent-level: 4; indent-tabs-mode: nil -*-
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2013-2017 Basho Technologies, Inc.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
 {minimum_otp_vsn, "R16"}.
 
-{erl_opts, [warnings_as_errors,
-            {parse_transform, lager_transform},
-            debug_info,
-            warn_untyped_record]}.
+{brt_protect, true}.
+
+{erl_opts, [
+    warn_export_all,
+    warn_export_vars,
+    warn_obsolete_guard,
+    warn_untyped_record,
+    warn_unused_import,
+    warnings_as_errors,
+    {parse_transform, lager_transform}
+]}.
 
-{deps, [getopt, lager]}.
+{escript_emu_args,
+    "%%! -escript main cuttlefish_escript +S 1 +A 0\n"}.
 
-{escript_emu_args, "%%! -escript main cuttlefish_escript +S 1 +A 0\n"}.
 {escript_incl_apps, [goldrush, getopt, lager, cuttlefish]}.
 {escript_main_app, cuttlefish}.
 
-{provider_hooks, [{post, [{compile, {default, escriptize}}]}]}.
+{provider_hooks, [
```

Comment:
the current solution to use neotoma only in the dev profilse seems better then hard including it